.travis.yml

```diff
@@ -6,36 +6,77 @@
 # I would prefer to test against what is in neurodebian
 language: python
 python:
-  - "2.7"
```

Comment:
Why do you remove 2.7? Is it causing trouble?
Yes, Python 2.7 is going EOL but it will be there for a while, even after 2020. I wouldn't drop it without compelling reason.

Reply:
Some of the versions of our dependencies do not support it anymore (specifically Scikit-learn). With new features introduced in versions, that do not support Python 2.7,  we require the dependencies to offer a way to address these backwards compatibility issues in new code. I was unsure, whether there will be a sensible way to work around this in general. Excluding Python 2.7 we could avoid the issue. If it is more reasonable on the other hand to include testing the most recent code on python 2.7 and older versions of our dependencies I can set it up. We still have the option to undo it, whenever we run into a specific case of this issue.

Reply:
Okay, thanks for the clarification. Would something speak against adding some [excludes](https://github.com/mdp-toolkit/mdp-toolkit/blob/master/.travis.yml#L23) for Python 2.7 and newer scikits? At least for a while?

Reply:
We could do that.